depends/packages/libnatpmp.mk

```diff
@@ -11,6 +12,10 @@ define $(package)_set_vars
   $(package)_build_env+=CFLAGS="$($(package)_cflags) $($(package)_cppflags)" AR="$($(package)_ar)"
```

Comment:
Can the ` $(package)_build_opts_darwin=LIBTOOL="$($(package)_libtool)"` line above (and same for miniupnpc) be removed now? I removed them and things compiled fine.

Reply:
Yep, thanks for catching this. Updated and squashed.